### PR TITLE
Learning Javascript woo

### PR DESCRIPTION
**Pull request by jaredstevick**

### CODE REVIEW

**Comment by jaredstevick** [comment]
Link to current WIP:
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/123

**Comment by johnalxndr** [comment]
:+1: !

**Comment by jrutishauser** [comment]
:+1:

**Comment by jgarcia99** [comment]
:+1:

**Comment by al-the-x** [comment]
Some more commits would be nice...

File: pub/css/main.css [hunk 1]
```diff
@@ -96,21 +96,109 @@ textarea {
...
+#E1:before {
+    position: absolute;
+    content: "\2654";
+}
 
+/*pawns*/
+#A2:before, #B2:before, #C2:before, #D2:before, #E2:before, #F2:before, #G2:before, #H2:before {
+    position: absolute;
+    content: "\2659";
+}
```
Comment (al-the-x) [inline pub/css/main.css:201]: Haven't gotten to your later submissions yet, but I hope you've come to a better solution now... adding pawns via `#id` references is the most tedious, for sure. Using a `.pawn` class would be more efficient.

File: pub/docs/js.md [hunk 2]
```diff
@@ -1,74 +1,251 @@
...
-### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
-
-* _symbol_: `+`
-* _pronunciation_: "plus", "added to"
-* _examples_:
-```javascript
-1 + 2 + 3 // 6, of course
-'1' + 2 + 3 // '123', obviously... WTF?
-```
+<<<<<<< HEAD
```
Comment (al-the-x) [inline pub/docs/js.md:1]: Looks like you committed a file "in conflict" here... Not sure what happened, but you need to remove these conflict markers _and_ the duplicate content. Selecting either `--ours` or `--theirs` when conflicts occur will make resolving conflicts easier... too bad `git status` doesn't show you which you've selected... or that you've selected _anything_. :/

File: pub/docs/js.md [hunk 3]
```diff
@@ -1,74 +1,251 @@
...
+```
+
+### [Primitive: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+* _symbol_: `true` and `false`
+* _pronunciation_: "True" and "False"
+* _examples_:
+```javascript
+'hello'==4 // false
+```
```
Comment (al-the-x) [inline pub/docs/js.md:20]: While the `==` operator _does_ evaluate to a Boolean value, it's not strictly a Boolean, itself. Examples of Boolean literals or values _converted_ to Booleans with the `Boolean()` wrapper (like you kinda did for `Number()` below) would be preferred.

File: pub/docs/js.md [hunk 4]
```diff
@@ -1,74 +1,251 @@
...
+
+### [Primitive: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+* _symbol_: `new Number(value);`
+* _pronunciation_: "number"
+* _examples_:
+```javascript
+Number("123") // 123
+```
```
Comment (al-the-x) [inline pub/docs/js.md:29]: Any other ways to write a `Number` value? Literals like `123` are fair game, as are the other literals we discussed in class today. Might be nice to document them for future use.

File: pub/docs/js.md [hunk 5]
```diff
@@ -1,74 +1,251 @@
...
+
+### [Primitive: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+* _symbol_: `"string"`
```
Comment (al-the-x) [inline pub/docs/js.md:33]: Don't forget those single quotes, too...!

File: pub/docs/js.md [hunk 6]
```diff
@@ -1,74 +1,251 @@
...
+```
+
+### [Literal: Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+* _symbol_: `object(x)`
+* _pronunciation_: "object(stuff)"
+* _examples_:
+```javascript
+var object = {} // creates an empty object with no properties
+```
```
Comment (al-the-x) [inline pub/docs/js.md:47]: So you got it right in your example... the _symbols_ used for Object literals are `{ }`... but how do I put something _inside_? There might be some more symbols that you need to document.

File: pub/docs/js.md [hunk 7]
```diff
@@ -1,74 +1,251 @@
...
+
+### [Literal: Regular Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)
+
+* _symbol_: ``
+* _pronunciation_:
+* _examples_:
+```javascript
+
+```
```
Comment (al-the-x) [inline pub/docs/js.md:65]: Totally not something we covered or expected by the assignment... Maybe a bonus later in the course.

File: pub/docs/js.md [hunk 8]
```diff
@@ -1,74 +1,251 @@
...
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+=======
```
Comment (al-the-x) [inline pub/docs/js.md:76]: Here's the _second_ conflict marker. Everything below is probably duped.

File: pub/index.html [hunk 9]
```diff
@@ -22,7 +23,115 @@
...
+        <td id="E1"> </td>
+        <td id="F1"> </td>
+        <td id="G1"> </td>
+        <td id="H1"> </td>
+    </tr>
+    </tbody>
+</table>
+</div>
+
+<!--end of container-->
```
Comment (al-the-x) [inline pub/index.html:134]: So about those classes...

File: pub/docs/js.md [hunk 10]
```diff
@@ -1,74 +1,251 @@
...
+```
+
+### [Global Object: Undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)
+
+* _symbol_: `undefined`
+* _pronunciation_: "undefined"
+* _examples_: 
+```javascript
+if var(x)=undefined{this code executes} else {this happens}
+```
```
Comment (al-the-x) [inline pub/docs/js.md:230]: Try running this through the REPL or `node` interpreter. JavaScript will not like you... Make sure your examples are real!

File: pub/docs/js.md [hunk 11]
```diff
@@ -1,74 +1,251 @@
...
+
+### [String Operator: Concatenation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Comparison_operators)
+
+* _symbol_: `+`
+* _pronunciation_: "concatenate"
+* _examples_:
+```javescript
+"cat + "dog" // "catdog" and "10" + 2 // 102
+```
```
Comment (al-the-x) [inline pub/docs/js.md:221]: This probably worked out better in your head.

File: pub/docs/js.md [hunk 12]
```diff
@@ -1,74 +1,251 @@
...
+
+### [Operator: Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+* _symbol_: `++`
+* _pronunciation_: `in cre ment`
+* _examples:
+```javascript
+4++ // 5, 6, 7....
+```
```
Comment (al-the-x) [inline pub/docs/js.md:168]: Actually, that won't work, either. Try it!

What _kind_ of operator is the increment?

**Pull request merged**